src/algebra/algebra/basic.lean

```diff
@@ -394,6 +394,9 @@ instance matrix_algebra (n : Type u) (R : Type v)
   smul_def' := by { intros, simp [matrix.scalar], },
   ..(matrix.scalar n) }
 
+@[simp] lemma matrix.algebra_map_eq_smul (n : Type u) {R : Type v} [decidable_eq n] [fintype n]
```

Comment:
I was thinking that maybe this should be the simp-normal form for all algebra maps. Is it worth asking on zulip?